Apps/phtrendmicroapexone/trendmicroapexone_consts.py

```diff
@@ -0,0 +1,43 @@
+# File: trendmicroapexone_consts.py
+#
+# Copyright (c) 2021 Splunk Inc.
+#
+# Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+
+APEX_ONE_PRODUCT_AGENTS_ENDPOINT = "/WebApp/API/AgentResource/ProductAgents"
+
+APEX_ONE_RESPONSE_EMPTY_MSG = "Empty result set from server. Please check your inputs"
+
+APEX_ONE_REPONSE_CODES = {
```

Comment:
@dfederschmidt, could you please check for this constant? It looks like that it is not being used anywhere in the connector. If we are not using it, we can remove it and if you want to keep it, then it's okay. Please confirm.

Reply:
Yes, this can be removed, it's a leftover from development because it turned out the API actually delivers the meaning of response codes on it's own. Sorry for the confusion